src/utils/ip.ts

```diff
@@ -40,3 +44,21 @@ export default function getIp(): string {
 
   return address
 }
+
+export async function getPublicIp() {
+  if (process.env.NODE_ENV === 'test') {
+    publicIpAddress = '127.0.0.1'
+    return
+  }
```

Comment:
it's better to not pollute our code with testing conditionals. Instead, the tests should mock this constant before each tests. For example, in test/index.test.ts

```javascript
// in test/index.test.ts
import sinon from 'sinon'
describe('monika', () => {
  let getPublicIPStub: any
  beforeEach(() => {
    getPublicIPStub = sinon.stub(Util, 'getPublicIp' as never)
    // @ts-ignore
    Util.publicIpAddress = '127.0.01'
  })
  afterEach(() => {
    getPublicIPStub.restore()
  })
```

Reply:
fixed. please recheck